File: src/main/java/com/gmail/borlandlp/minigamesdtools/events/sponge/INITIALIZATION_EVENT.java
package com.gmail.borlandlp.minigamesdtools.events.sponge;

import org.bukkit.event.Event;
import org.bukkit.event.HandlerList;

/*
* Пул конфигов загружен
* API не инициализировано
* Стандартный набор создателей инициализирован
* */
public class INITIALIZATION_EVENT extends Event {
    private static final HandlerList handlers = new HandlerList();

    @Override
    public HandlerList getHandlers() {
        return handlers;
    }

    public static HandlerList getHandlerList() {
        return handlers;
    }
}

File: src/main/kotlin/com/gmail/borlandlp/minigamesdtools/events/sponge/INITIALIZATION_EVENT.kt
package com.gmail.borlandlp.minigamesdtools.events.sponge

import org.bukkit.event.Event
import org.bukkit.event.HandlerList

/*
* Пул конфигов загружен
* API не инициализировано
* Стандартный набор создателей инициализирован
* */
class INITIALIZATION_EVENT : Event() {
    override fun getHandlers(): HandlerList {
        return handlerList
    }

    companion object {
        @JvmStatic
        val handlerList = HandlerList()
    }
}